backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Migration/v0.07-authorization-admin/02-setup-procedures.sql

```diff
@@ -37,7 +37,48 @@ AS $BODY$
     _performedByUserId,
     _blobStoragePolicyPath,
     _blobStorageVersionId,
-    _resourceid,
-    _resourcetype
+	_resourceid,
+	_resourcetype
   ) RETURNING *;
 $BODY$;
+
+ALTER FUNCTION delegation.insert_delegationchange(delegation.delegationchangetype, character varying, integer, integer, integer, integer, character varying, character varying, character varying, character varying)
+    OWNER TO platform_authorization_admin;
```

Comment:
Redundant? platform_authorization_admin is the user running these script and creates the database artifacts. 
If, anything you might need to run grant script for new functions to the runtime platform_authorization user